APIAlfresco.php

```diff
@@ -79,13 +93,16 @@ public function checkResponse()
      */
     public function setFolderByPath($folder, array $options = array())
     {
-        $obj = $this->repository->getObjectByPath($folder, $options);
-        $propiedad = $obj->properties['cmis:baseTypeId'];
-        if ($propiedad != 'cmis:folder') {
-            echo 'The object is not a folder';
-            exit(255);
-        } else {
-            $this->parentFolder = $obj;
+        try {
+            $obj = $this->repository->getObjectByPath($folder, $options);
+            $properties = $obj->properties['cmis:baseTypeId'];
+            if ($properties != 'cmis:folder') {
+                throw new Exception('The object is not a folder');
```

Comment:
Might be better use different kind of exceptions, e.g:

UnexpectedValueException 

http://php.net/manual/en/class.unexpectedvalueexception.php